issues with the docker stuff in the app-service-template

When we try to follow the instructions in `https://github.com/edgexfoundry-holding/app-functions-sdk-python/tree/main/app-service-template`, 
we run into the following error

```
make run no-secty ds-virtual mqtt-bus keeper
echo MQTT_VERBOSE=
MQTT_VERBOSE=
docker compose  -p edgex -f docker-compose-base.yml -f add-device-virtual.yml -f add-postgres.yml -f add-mqtt-messagebus.yml -f add-mqtt-broker-mosquitto.yml convert --output docker-compose.yml.gen
if [ "0" = "0" ]; then rm -rf ./gen_ext_compose; fi
echo "# Generated with: Docker Compose version v2.29.7-desktop.1" > docker-compose.yml
cat docker-compose.yml.gen >> docker-compose.yml
rm docker-compose.yml.gen
docker compose  -p edgex up -d mqtt-bus keeper
no such service: mqtt-bus
make: *** [Makefile:1063: run] Error 1
```

Upon a little inspection, we found that there is no mqtt-bus in the compose file. We tried replacing mqtt-bus with mqtt-broker, it didn't work either. 
Any suggestions?

--- a/app-service-template/README.md
+++ b/app-service-template/README.md
@@ -7,7 +7,7 @@ This sample application service--**app-new-service**--is configured with default
 Follow the instructions below to try this sample application service:
 
 1. Prepare your local development environment with following dependencies:
-   - python 3.10 or above
+   - python 3.10
    - docker
    - make
    - git
@@ -15,20 +15,20 @@ Follow the instructions below to try this sample application service:
 2. As the app service will use default Message Bus trigger to consume events from EdgeX MessageBus, we have to run up EdgeX core services and device-virtual service to simulate the EdgeX events in your local environment. The easiest way to run up EdgeX services is through script available from https://github.com/edgexfoundry/edgex-compose:
    - `git clone https://github.com/edgexfoundry/edgex-compose.git`
    - switch into the `edgex-compose/compose-builder` directory
-   - `make run no-secty ds-virtual mqtt-bus keeper`
+   - `make run no-secty ds-virtual`
 
 3. By the completion of above step, you will have EdgeX core services, device-virtual service, and an MQTT broker running as docker container in your local environment, and you can examine their running status by using command: `docker ps -a`. Note that the running device-virtual service will periodically generate EdgeX events and publish them to EdgeX message bus, and you can observe those EdgeX events by using following command:
    - `docker exec edgex-mqtt-broker mosquitto_sub -v -t 'edgex/events/#'`
 
 4. Now we have EdgeX services up and running, the next step is to copy contents of this repository to your local environment by using following command:
-   - `git clone https://github.com/IOTechSystems/app-functions-sdk-python.git`
+   - `git clone https://github.com/edgexfoundry/app-functions-sdk-python.git`
 
 5. Switch into the `app-functions-sdk-python` directory
 
 6. Examine the code of `app-service-template/main.py` to understand how to write a custom application service using the App Functions Python SDK.
 
-7. Create a virtual environment in the root of the repository in your local environment:
-   - `python3 -m venv venv`
+7. Create a virtual environment based on python3.10 in the root of the repository in your local environment:
+   - `python3.10 -m venv venv`
 
 8. Switch to the virtual environment:
    - `source ./venv/bin/activate`
@@ -45,5 +45,10 @@ Follow the instructions below to try this sample application service:
 
 12. Once the app-new-service is running, it will consume EdgeX events from the EdgeX message bus, filter for events coming from **Random-Integer-Device**, and then re-publish the events to an external MQTT broker **test.mosquitto.org** via **test_topic** topic. To observe those events being published to **test.mosquitto.org**, you can use following command:
     - `docker exec edgex-mqtt-broker mosquitto_sub -v -t 'test_topic' -h test.mosquitto.org`
+    ```shell
+    test_topic {"id": "247a70e3-2aa1-42be-88e5-82d6e9bf1041", "deviceName": "Random-Integer-Device", "profileName": "Random-Integer-Device", "sourceName": "Int8", "origin": 1738569195201797427, "readings": [{"id": "37286bf5-82f5-45bb-b93b-db1ff64a6644", "origin": 1738569195201797427, "deviceName": "Random-Integer-Device", "resourceName": "Int8", "profileName": "Random-Integer-Device", "valueType": "Int8", "value": "-2", "units": "", "binaryValue": null, "objectValue": null, "tags": {}, "mediaType": ""}], "tags": {}}
+    test_topic {"id": "bbe07061-86e2-4703-bc85-a471bd03ec31", "deviceName": "Random-Integer-Device", "profileName": "Random-Integer-Device", "sourceName": "Int16", "origin": 1738569195201861388, "readings": [{"id": "2c59a79b-8d37-4f5a-8267-99902ab4b6d3", "origin": 1738569195201861388, "deviceName": "Random-Integer-Device", "resourceName": "Int16", "profileName": "Random-Integer-Device", "valueType": "Int16", "value": "-2295", "units": "", "binaryValue": null, "objectValue": null, "tags": {}, "mediaType": ""}], "tags": {}}
+    test_topic {"id": "4de8c1a9-08fa-4a95-935f-7c0fa81cbca9", "deviceName": "Random-Integer-Device", "profileName": "Random-Integer-Device", "sourceName": "Int32", "origin": 1738569195201918924, "readings": [{"id": "2bf5e926-bbe3-4468-91e0-0e8d2cb6d67c", "origin": 1738569195201918924, "deviceName": "Random-Integer-Device", "resourceName": "Int32", "profileName": "Random-Integer-Device", "valueType": "Int32", "value": "509211916", "units": "", "binaryValue": null, "objectValue": null, "tags": {}, "mediaType": ""}], "tags": {}}
+    ```
 
 13. To stop the app-new-service, you can press `Ctrl+C` to stop the running process.

